### PR TITLE
fix(salesforce): call api failing

### PR DESCRIPTION
**Pull request by davidvitora**
The call API action would fail (silently) if there is no body in the return from the axios call

I also exposed the status code

### CODE REVIEW

File: integrations/salesforce/src/misc/utils/apiUtils.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const makeRequest = async (
   input: Input,
   accessToken: string
 ): Promise<AxiosResponse> => {
-  return await axios({
+  return axios({
```
Comment (ermek-botpress) [inline integrations/salesforce/src/misc/utils/apiUtils.ts:9]: Why was await removed?

Comment (davidvitora) [reply integrations/salesforce/src/misc/utils/apiUtils.ts:9]: await is not really necessary if you are returning the promise in an async call, no issues were caused by this but I removed the redundance, do you want to keep it?

Comment (ermek-botpress) [reply integrations/salesforce/src/misc/utils/apiUtils.ts:9]: No, I was just curious

**Review by ermek-botpress — APPROVED** [review]
LGTM

**Pull request merged**